main.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ConfigSettings struct {
+	Port  string   `yaml:"port"`
+	Repos []string `yaml:"repos"`
+}
+
+// Global Configuration for the service
```

Comment:
You should use proper go doc syntax here
```suggestion
// Config holds the global configuration for the service
```